store/postgres/src/relational/ddl_tests.rs

```diff
@@ -60,6 +60,75 @@ fn check_eqv(left: &str, right: &str) {
     }
 }
 
+#[test]
+fn test_manual_index_creation_ddl() {
+    let layout = Arc::new(test_layout(BOOKS_GQL));
+
+    fn assert_generated_sql(
```

Comment:
If you add a `#[track_caller]` before the `fn`, failures will point to the line where this was called from, which can help with fixing failing tests.